bemani/format/iidxmusicdb.py

```diff
@@ -80,6 +80,9 @@ def __parse_db(self, data: bytes) -> None:
         elif data[4] == 0x18:
             offset = 0xc360
             leap = 0x340
+        elif data[4] == 0x1a:
```

Comment:
This is such a gross hack on my part. This number is clearly usable for calculating offsets, but oh well.